include/openssl/evp.h

```diff
@@ -462,6 +462,11 @@ typedef int (EVP_PBE_KEYGEN) (EVP_CIPHER_CTX *ctx, const char *pass,
 int EVP_MD_type(const EVP_MD *md);
 # define EVP_MD_nid(e)                   EVP_MD_type(e)
 const char *EVP_MD_name(const EVP_MD *md);
+int EVP_MD_number(const EVP_MD *md);
+int EVP_MD_is_a(const EVP_MD *md, const char *name);
+void EVP_MD_doall_names(const EVP_MD *md,
```

Comment:
There is inconsistency between the `doall_names` functions and the other old `do_all_...` functions. There should be underscore between do and all. Or, if the inconsistency is there because the do_all_... and doall_names have quite different purpose I'd suggest to indicate it by other means. Such as by using `EVP_MD_names_do_all` or so.

Reply:
Sure, I can go with either. Which would you prefer?

Reply:
I think I'll go with `EVP_FOO_names_do_all`

Reply:
Done